Sources/TZSettingsView/TZSettingsCell/TZSettingsCellConfiguration.swift

```diff
@@ -65,16 +72,31 @@ extension TZSettingsView {
          The hint to be applied to the cell, if any.
          */
         var hint: String?
+        /**
+         The description text to be displayed in the cell, if any.
+         */
+        var description: String?
```

Comment:
Why is the description optional while the cell type should be a cell with a description?

Reply:
I deleted the cell type description with action since it matches the action cell , so I kept the description this way in case we don't need to use it